app/models/immunisation_import_row.rb

```diff
@@ -525,7 +525,7 @@ def validate_clinic_name
           "is greater than #{MAX_FIELD_LENGTH} characters long"
         )
       elsif clinic_name_required &&
-            !organisation.community_clinics.exists?(name: clinic_name.to_s)
+            !organisation.community_clinics.any? { |clinic| clinic.name.downcase == clinic_name.to_s.downcase }
```

Comment:
```suggestion
            !organisation.community_clinics.exists?("LOWER(name) = ?", clinic_name.to_s.downcase)
```

This should have the same behaviour but it's a more optimised query. In the original change, Rails will end up fetching all the community clinic records and creating an instance of them in memory to then call `.any?` on, whereas using `exists?` means we'll execute a single `SELECT EXISTS` SQL query.